releasenotes/notes/remove-device_name-populateDeviceField-3967de244ebed07f.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Removed consideration of the ``device_name`` tag from the populateDeviceField logic in series.
```

Comment:
Would something like `"The ``device_name`` tag is not used anymore to populate the ``Device`` field of a series. Only the ``device`` tag is considered."` be easier to understand for a user? Not sure how precise we want to be in the release note.

Reply:
I don't know either. I think your sentence is clearer. Thanks man.